frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -1289,6 +1290,15 @@ export const eventUsageLogic = kea<eventUsageLogicType>([
                 end_date: survey.end_date,
             })
         },
+        reportSurveyWithBranchingCreated: ({ survey }) => {
+            posthog.capture('survey with branching created', {
```

Comment:
instead of a separate event, I think I'll prefer this as a property on the survey created / edited events: We already track other features on that event, can add this there too

Reply:
much nicer 👍